src/types.ts

```diff
@@ -70,3 +83,5 @@ export type VerifySDJWT = (
   sdJWT: string,
   { getIssuerKey, expected_aud, expected_nonce }: VerifySdJwtOptions,
 ) => Promise<SDJWTPayload>;
+
+export type IssueSDJWT = (options: IssueSDJWTOptions) => Promise<string>;
```

Comment:
@ragnika so this could be something like this:
```
export type IssueSDJWT = (
  header: JWTHeaderParameters, 
  payload: JWTPayload, 
  disclosures: DisclosureType, 
  opts: IssueSDJWTOptions
) => Promise<string>;
```